etna/packages/etna-js/utils/dataframe.ts

```diff
@@ -0,0 +1,171 @@
+import {HyperFormula} from 'hyperformula';
```

Comment:
Shared between `archimedes-node` and `Vulcan`, so consolidating code in `etna-js`